<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="idea-PC" failures="2" tests="6" name="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" time="157.001" errors="0" timestamp="2022-05-11T07:04:36 IST" skipped="4">
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainWithProductTest" time="20.725">
    <skipped/>
  </testcase> <!-- createCampainWithProductTest -->
  <system-out/>
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainTest" time="19.115">
    <failure message=" is not created and verification --&gt;fail expected [TestYantra Hackathon Campaign9516] but found [[ CAM65 ] TestYantra Hackathon Campaign9516 - Campaign Information]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError:  is not created and verification -->fail expected [TestYantra Hackathon Campaign9516] but found [[ CAM65 ] TestYantra Hackathon Campaign9516 - Campaign Information]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest.createCampainTest(CreatingCampaignTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createCampainTest -->
  <system-out/>
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainWithProductTest" time="21.292">
    <skipped/>
  </testcase> <!-- createCampainWithProductTest -->
  <system-out/>
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainWithProductTest" time="53.608">
    <skipped/>
  </testcase> <!-- createCampainWithProductTest -->
  <system-out/>
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainWithProductTest" time="21.481">
    <skipped/>
  </testcase> <!-- createCampainWithProductTest -->
  <system-out/>
  <testcase classname="com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest" name="createCampainWithProductTest" time="20.780">
    <failure message="BHEESHMA T 90 TANK3771 expected:&lt;BHEESHMA T 90 TANK3771&gt; but was:&lt;Selected product is not verified ++.FAIL&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: BHEESHMA T 90 TANK3771 expected:<BHEESHMA T 90 TANK3771> but was:<Selected product is not verified ++.FAIL>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest.createCampainWithProductTest(CreatingCampaignTest.java:101)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createCampainWithProductTest -->
  <system-out/>
</testsuite> <!-- com.crm.comcast.CampaignsTest.Test.CreatingCampaignTest -->
